worlds/alttp/docs/multiworld_es.md

```diff
@@ -1,223 +1,112 @@
-   **El archivo esta localizado en la sección "assets" en la parte inferior de la información de versión**. Si tu
-   intención es jugar la versión normal de multiworld, necesitarás el archivo `Setup.Archipelago.exe`
-    - Si estas interesado en jugar la variante que aleatoriza las puertas internas de las mazmorras, necesitaras bajar '
-      Setup.BerserkerMultiWorld.Doors.exe'
-    - Durante el proceso de instalación, se te pedirá donde esta situado tu archivo ROM japonés v1.0. Si ya habías
-      instalado este software con anterioridad y simplemente estas actualizando, no se te pedirá la localización del
-      archivo una segunda vez.
-    - Puede ser que el programa pida la instalación de Microsoft Visual C++. Si ya lo tienes en tu ordenador (
-      posiblemente por que un juego de Steam ya lo haya instalado), el instalador no te pedirá su instalación.
-
-2. Si estas usando un emulador, deberías asignar la versión capaz de ejecutar scripts Lua como programa por defecto para
-   lanzar ficheros de ROM de SNES.
-    1. Extrae tu emulador al escritorio, o cualquier sitio que después recuerdes.
-    2. Haz click derecho en un fichero de ROM (ha de tener la extensión sfc) y selecciona **Abrir con...**
-    3. Marca la opción **Usar siempre esta aplicación para abrir los archivos .sfc**
-    4. Baja hasta el final de la lista y haz click en la opción **Buscar otra aplicación en el equipo** (Si usas Windows
-       10 es posible que debas hacer click en **Más aplicaciones**)
-    5. Busca el archivo .exe de tu emulador y haz click en **Abrir**. Este archivo debe estar en el directorio donde
-       extrajiste en el paso 1.
-
-### Instalación en Macintosh
-
-- ¡Necesitamos voluntarios para rellenar esta seccion! Contactad con **Farrak Kilhn** (en inglés) en Discord si queréis
-  ayudar.
-
-## Configurar tu archivo YAML
-
-### Que es un archivo YAML y por qué necesito uno?
-
-Tu archivo YAML contiene un conjunto de opciones de configuración que proveen al generador con información sobre como
-debe generar tu juego. Cada jugador en una partida de multiworld proveerá su propio fichero YAML. Esta configuración
-permite que cada jugador disfrute de una experiencia personalizada a su gusto, y cada jugador dentro de la misma partida
-de multiworld puede tener diferentes opciones.
-
-### Donde puedo obtener un fichero YAML?
-
-La página "[Generate Game](/games/A%20Link%20to%20the%20Past/player-settings)" en el sitio web te permite configurar tu
-configuración personal y descargar un fichero "YAML".
-
-### Configuración YAML avanzada
-
-Una version mas avanzada del fichero Yaml puede ser creada usando la pagina ["Weighted settings"](/weighted-settings),
-la cual te permite tener almacenadas hasta 3 preajustes. La pagina "Weighted Settings" tiene muchas opciones
-representadas con controles deslizantes. Esto permite elegir cuan probable los valores de una categoría pueden ser
-elegidos sobre otros de la misma.
```

Comment:
This section should also stay in, accepting main's version of the translation and making any updates on that.